account_payment_order/models/account_move.py

```diff
@@ -24,7 +24,6 @@ class AccountMove(models.Model):
     reference_type = fields.Selection(
         selection=[("none", "Free Reference"), ("structured", "Structured Reference")],
         readonly=True,
-        states={"draft": [("readonly", False)]},
```

Comment:
Isn't possible to define this with new syntax `readonly="state != 'draft'"` and don't need to put it in each view?